purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -0,0 +1,103 @@
+        this.dangerousSettings = builder.dangerousSettings
+    }
+
+    open class Builder(
+        @get:JvmSynthetic internal val context: Context,
+        @get:JvmSynthetic internal val apiKey: String,
+        @get:JvmSynthetic internal val appUserID: String,
+    ) {
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var observerMode: Boolean = false
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var service: ExecutorService? = null
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var store: Store = Store.PLAY_STORE
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var diagnosticsEnabled: Boolean = false
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var verificationMode: EntitlementVerificationMode = EntitlementVerificationMode.default
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var dangerousSettings: DangerousSettings = DangerousSettings(customEntitlementComputation = true)
+
+        fun observerMode(observerMode: Boolean) = apply {
+            this.observerMode = observerMode
+        }
+
+        fun service(service: ExecutorService) = apply {
+            this.service = service
+        }
+
+        fun store(store: Store) = apply {
+            this.store = store
+        }
+
+        /**
+         * Enabling diagnostics will send some performance and debugging information from the SDK to our servers.
+         * Examples of this information include response times, cache hits or error codes.
+         * This information will be anonymized so it can't be traced back to the end-user.
+         * The default value is false.
+         */
+        fun diagnosticsEnabled(diagnosticsEnabled: Boolean) = apply {
+            this.diagnosticsEnabled = diagnosticsEnabled
+        }
+
+        /**
+         * Sets the [EntitlementVerificationMode] to perform signature verification of requests to the
+         * RevenueCat backend.
+         *
+         * When changing from [EntitlementVerificationMode.DISABLED] to other modes, the SDK will clear the
+         * CustomerInfo cache.
+         * This means users will need to connect to the internet to get their entitlements back.
+         *
+         * The result of the verification can be obtained from [EntitlementInfos.verification] or
+         * [EntitlementInfo.verification].
+         *
+         * Default mode is disabled. Please see https://rev.cat/trusted-entitlements for more info.
+         */
+        fun entitlementVerificationMode(verificationMode: EntitlementVerificationMode) = apply {
+            this.verificationMode = verificationMode
+        }
+
+        /**
+         * Only use a Dangerous Setting if suggested by RevenueCat support team.
+         */
+        fun dangerousSettings(dangerousSettings: DangerousSettings) = apply {
+            this.dangerousSettings = dangerousSettings.copy(customEntitlementComputation = true)
```

Comment:
Note how I'm doing this, so in case somone using this package changes dangerous settings, we still enable customEntitlementComputation.

Reply:
Hmm the `customEntitlementComputation` in `DangerousSettings` is internal, so devs won't be able to set it up themselves. So it should always be `false` in the `defaults` flavor and true in the `customEntitlementComputation` flavor. Not sure if I'm missing something?

Reply:
Oh then nevermind, I had missed that point